alcs-frontend/src/app/shared/drag-drop-board/drag-drop-item.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface DragDropItem {
```

Comment:
not in scope of this PR, I wonder if we will need to capture who moved item between columns?

Reply:
I think we will, having a last updated by is useful for Applications. We should check with product if they want a full history a la Jira style or just tracking the last person to make changes.